### ISSUE TITLE
Badly formatted/improperly handled CSV export

### ISSUE THREAD

**Issue by mnapolitano89** [churchlab/millstone#600]
For example, I have a line like:

(HIGH||-|-0|231|minC||CODING|b1176|1|1)']

in there

**Comment by glebkuznetsov** [comment]
Is the problem that it fails to open in excel?

**Comment by mnapolitano89** [comment]
no, just that some rows are missing because of bad formatting. A lot of the individual keys are also messed up
